src/DynamoCoreWpf/Controls/ShortcutToolbar.xaml.cs

```diff
@@ -59,6 +60,14 @@ public ShortcutToolbar(DynamoViewModel dynamoViewModel)
             else {
                 logoutOption.Visibility = Visibility.Collapsed;
             }
+
+            this.Loaded += ShortcutToolbar_Loaded;
+        }
+
+        private void ShortcutToolbar_Loaded(object sender, RoutedEventArgs e)
+        {
+            IsSaveButtonEnabled = false;
+            IsExportMenuEnabled = false;
```

Comment:
If the initial state of these controls are already disabled, why disable these again?

Reply:
I guess you are referring to the MenuItems, yes they are disabled in the xaml side, the above code is for the Toolbar buttons